apps/web/src/data/en/index.ts

```diff
@@ -795,6 +795,32 @@ export const loggedInData = {
           maxWidthPlaceholder: 'Enter the maximum width',
           helpTooltipText: 'More information and help related to Image Plugin',
           change: 'Change image',
+          licenceFree: 'Licence Free Images',
+          pixabayText: 'Pictures will be fetched from Pixabay', // TODO: fix text
+          pixabayLoadedText: "These pictures were loaded from Pixabay's API.", // TODO: fix text
+          searching: 'Searching for images ...',
+          loadingImage: 'Downloading image ...', // "Bild wird heruntergeladen..."
+          noImagesFound: 'No images found',
+          searchTags: {
+            math: 'Mathematikunterricht',
+            chemistry: 'Chemieunterricht',
+            physics: 'Physikunterricht',
+            biology: 'Biologiediagramme',
+            tech: 'Technologie in der Schule',
+            informatik: 'Informatikkurs',
+            geometry: 'Geometrische Figuren',
+            experiments: 'Experimente',
+            science: 'Naturbeobachtungen',
+            class: 'Klassenzimmer',
+            books: 'Schulbücher',
+            hof: 'Schulhof',
+            teaching: 'Lehrmaterialien',
+            board: 'Schultafel',
+            groupWork: 'Gruppenarbeit',
+            projects: 'Schulprojekte',
+            games: 'Lernspiele',
+            creative: 'Kreatives Lernen',
```

Comment:
since those suggestions might change more often I again wonder if they should live in the i18n strings or if we should hardcode them. Not sure.

Reply:
Will keep them here for now since they will need to be translated. Have taken a note to discuss with Gregor